config/sync/system.site.yml

```diff
@@ -12,3 +12,4 @@ langcode: en
 default_langcode: en
 _core:
   default_config_hash: yTxtFqBHnEWxQswuWvkjE8mKw2t8oKuCL1q8KnfHuGE
+mail_notification: ''
```

Comment:
What's this for?

Reply:
this happened when I enabled the email verification.